cmd/neofs-node/meta_new.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/config"
+	"github.com/nspcc-dev/neo-go/pkg/core/storage/dbconfig"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/fixedn"
+	"github.com/nspcc-dev/neofs-node/pkg/services/sidechain"
+	"go.uber.org/zap"
+)
+
+func initMeta_new(c *cfg) {
+	l := c.log.With(zap.String("component", "metadata chain"))
+
+	v, err := c.cfgMorph.client.GetVersion()
+	fatalOnErr(err)
+
+	fsChainProtocol := v.Protocol
+	standByCommittee := make([]string, 0, len(v.Protocol.StandbyCommittee))
+	for _, c := range v.Protocol.StandbyCommittee {
+		standByCommittee = append(standByCommittee, c.StringCompressed())
+	}
+
+	p2pCfg := c.appCfg.Meta.P2P
+
+	var chainCfg = config.Config{
+		ProtocolConfiguration: config.ProtocolConfiguration{
```

Comment:
Just for the record: I don't see `Hardforks` being set. It means that all *stable* hardforks are active starting from genesis. It's expected and correct behaviour for meta chain, I'm just letting you know that.

However, NeoGo is continuously adding more hardforks, so if some new hardfork is added, then it may affect states after subsequent NeoGo dependency update in meta chain. So to me, it's better to explicitly set all desired hardfork heights to 0 here.